tokens.json

```diff
@@ -1485,5 +1485,14 @@
         "symbol": "USDT",
         "account": "issue.newdex",
         "chain": "eos"
-    }
+    },
+    {
+    "name": "<Tatch Capital Note>",
+    "logo": "https://github.com/EdwardTatch/eos-airdrops/blob/master/logos/TCN-Tatch-transp.png",
+    "logo_lg": "https://github.com/EdwardTatch/eos-airdrops/blob/master/logos/TCN-Tatch-HQ.png",
+    "symbol": "TCN",
+    "account": "tatchcapital",
```

Comment:
Wrong token account name.

I think it's `tcapitalnote` correct?

Reply:
Hi Denis, 


Yes indeed, tcapitalnote is the correct account. tatchcapital is the issuer, my bad.

With best regards, 

Cédric